01week/resume/bio.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="resumesheet.css" type="text/css">
+    <title>Bio - Cameron Harmon</title>
+</head>
+
+<body>
+    <nav>
+        <a href="home.html">Home</a> |
+        <a href="bio.html">Bio</a> |
+        <a href="resume.html">Resume</a> |
+        <a href="contact.html">Contact</a>
+    </nav>
+    <h1>Learning to Code in the heart of Texas.</h1>
+    <img id="photo" src="me.jpg" alt="Profile Photo">
+    <p>Meet Cameron. Cameron grew up in Dallas, TX. After graduating from Texas State University San Marcos in 2012, Cameron moved to Austin in 2014. Cameron reads a book a week, is a student at Austin Code Academy, and prefers single speed if anybobdy's asking. Like many Austinites, Cameron became an instant taco aficionado. His goal is to learn everything he can about coding and become an Overlord of Webmastering. </p>
```

Comment:
Try to keep single lines of code from extending too far. You can create a new line for them once they get too large without changing how the string is displayed.